### ISSUE TITLE
Bibliography: research and catalog academic articles that implemented eye tracking solutions

### ISSUE THREAD

**Issue by KarinePistili** [ruxailab/web-eye-tracker#22]
We would like to have a bibliography of the state of the art for eye trackers. Catalog the papers you've read in a spreadsheet with title, year of publication, authors, abstract and an overview of the methods and results they achieve from your point of view.

**Comment by LostSputnik** [comment]
I want to work on this. I already have organized a number of papers from when I was writing the proposal, plus this integrates well with issue #17 , as they go hand in hand

**Comment by KarinePistili** [comment]
Sure! I will assign it you @LostSputnik

**Comment by sitamgithub-MSIT** [comment]
@KarinePistili I think this is helpful for all the proposals to prepare such a sheet so that we can track all our research papers and findings.

**Comment by KarinePistili** [comment]
True, all project ideas had some research background. Maybe we can do something like that for the other projects more in the future

**Comment by divyanshi-shukla24** [comment]
We would like to have a bibliography of the state of the art for eye trackers. Catalog the papers you've read in a spreadsheet with title, year of publication, authors, abstract and an overview of the methods and results they achieve from your point of view.
